### DISCUSSION TITLE
How to use open-props with Storybook?

### DISCUSSION THREAD

**Discussion by Akshat42** [argyleink/open-props#519] [Q&A]
I am trying to use open-props in Storybook. while importing it in preview.ts, I am getting lots of postCSS errors.

**Comment by argyleink** [comment]
I can help 👍🏻 
- are you using NPM?
- what are the errors?

my guess is that the module resolution system is trying to grab Open Props JS variables and not just importing the CSS. but maybe you want the JS variables?
